apps/api/src/app/analytics/analytics.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { AnalyticsService } from '@novu/application-generic';
+
+@Controller({
+  path: 'analytics',
+})
+export class AnalyticsController {
+  constructor(private analyticsService: AnalyticsService) {}
+
+  @Post('/telemetry')
```

Comment:
Should we add auth guard here and use that userId instead?

Reply:
Yes we definitely should. Otherwise, anyone could register fake events.